src/styles/index.scss

```diff
@@ -1,3 +1,102 @@
+@import url("./star.scss");
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  line-height: 18px;
+  color: #060b35;
+
+  --color-blue: #00acdc;
+}
+
+h2 {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  &__code {
+    color: #616070;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+    margin-top: 0;
```

Comment:
you can just reset default margin for some specific tags to avoid this ` margin-top: 0;`